test/benchmark/benchmark_lazy_loadable_test.rb

```diff
@@ -0,0 +1,65 @@
+require 'test_helper'
```

Comment:
Note: This is a test used to produce the benchmarks. It will be removed before merging.

Reply:
Please go ahead and remove this now -- I've reviewed this PR.